src/test/scala/com/microsoft/hyperspace/index/rankers/JoinIndexRankerTest.scala

```diff
@@ -31,57 +30,64 @@ class JoinIndexRankerTest extends SparkFunSuite {
   val t2c2 = AttributeReference("t2c2", StringType)()
 
   test("testRank: Prefer equal-bucket index pairs over unequal-bucket") {
-    val l_10 = index("l1", Seq(t1c1), Seq(t1c2), 10)
-    val l_20 = index("l2", Seq(t1c1), Seq(t1c2), 20)
-    val r_20 = index("r1", Seq(t2c1), Seq(t2c2), 20)
+    val l_10 = RankerTestHelper.index("l1", Seq(t1c1), Seq(t1c2), numBuckets = 10)
+    val l_20 = RankerTestHelper.index("l2", Seq(t1c1), Seq(t1c2), numBuckets = 20)
+    val r_20 = RankerTestHelper.index("r1", Seq(t2c1), Seq(t2c2), numBuckets = 20)
 
     val indexPairs = Seq((l_10, r_20), (l_20, r_20))
 
     val expectedOrder = Seq((l_20, r_20), (l_10, r_20))
-    val actualOrder = JoinIndexRanker.rank(indexPairs)
+    val actualOrder = JoinIndexRanker.rank(indexPairs, false)
     assert(actualOrder.equals(expectedOrder))
   }
 
   test("testRank: Prefer higher number of buckets if multiple equal-bucket index pairs found") {
-    val l_10 = index("l1", Seq(t1c1), Seq(t1c2), 10)
-    val l_20 = index("l2", Seq(t1c1), Seq(t1c2), 20)
-    val r_10 = index("r1", Seq(t2c1), Seq(t2c2), 10)
-    val r_20 = index("r2", Seq(t2c1), Seq(t2c2), 20)
+    val l_10 = RankerTestHelper.index("l1", Seq(t1c1), Seq(t1c2), 10)
+    val l_20 = RankerTestHelper.index("l2", Seq(t1c1), Seq(t1c2), 20)
+    val r_10 = RankerTestHelper.index("r1", Seq(t2c1), Seq(t2c2), 10)
+    val r_20 = RankerTestHelper.index("r2", Seq(t2c1), Seq(t2c2), 20)
 
     val indexPairs = Seq((l_10, r_10), (l_10, r_20), (l_20, r_20))
 
     val expectedOrder = Seq((l_20, r_20), (l_10, r_10), (l_10, r_20))
-    val actualOrder = JoinIndexRanker.rank(indexPairs)
+    val actualOrder = JoinIndexRanker.rank(indexPairs, false)
```

Comment:
nit: for readability

```suggestion
    val actualOrder = JoinIndexRanker.rank(indexPairs, hybridScanEnabled = false)
```